meson.build

```diff
@@ -127,7 +127,12 @@ nnstreamer_conf.set('TRIX_ENGINE_ALIAS', get_option('trix-engine-alias'))
 add_project_arguments('-DNNSTREAMER_CONF_FILE="' + join_paths(nnstreamer_inidir, 'nnstreamer.ini') + '"', language: 'c')
 
 # Dependencies
-glib_dep = dependency('glib-2.0')
+glib_dep = dependency('glib-2.0', version: '>= 2.68.0', required: false)
+if (glib_dep.found())
+  add_project_arguments('-DGLIB_USE_G_MEMDUP2', language: ['c', 'cpp'])
+else
+  glib_dep = dependency('glib-2.0')
```

Comment:
how about using macro in header file or version_compare() in meson

Use glib macro in header
```
#if GLIB_CHECK_VERSION (2, 68, 0)
#define _g_memdup g_memdup2
#else
#define _g_memdup g_memdup
#endif
```

or meson.build
```
if glib_dep.version().version_compare('>= 2.68.0')
  add_project_arguments('-DGLIB_USE_G_MEMDUP2', language: ['c', 'cpp'])
endif
```

IMO, 1st one is better.

Reply:
I am fine with dropping meson.build change and go for  GLIB_CHECK_VERSION() check in the header file if all agree with that.

However doing so requires addition of
`
#include <glib.h>
`
at the top of the header file so that the macro is guaranteed to be defined.
May not be a bid deal but my understanding is that at the moment .c/.cc files include glib.h explicitly, not indirectly via other header files.

Reply:
The first approach (no changes in meson) looks more beautiful, but we cannot ensure the order of ```#include``` statements; the coding-style checks (gst-indent, clang-format, ...) often force to reorder them and having such implicit conditions don't feel right.

Let's proceed with the second approach (adding a few lines in meson.build).

Reply:
Patchset updated with suggestion from @jaeyun-jung for option 2 in meson.build:

`
if glib_dep.version().version_compare('>= 2.68.0')
  add_project_arguments('-DGLIB_USE_G_MEMDUP2', language: ['c', 'cpp'])
endif
`